### DISCUSSION TITLE
startup sessions config issue

### DISCUSSION THREAD

**Discussion by kporkka** [kovidgoyal/kitty#3274] [Q&A]
Newbie here, just cannot figure this out. In my startup session config file, if i include a launch program command (e.g. launch htop), kitty opens, launches the program and when I exit the program (e.g. htop), then kitty also exits, which I don't want. Feeling stupid, please help me out. -kimmo

start.conf file:
# startup commands
launch htop

**Comment by Luflosi** [comment]
Does something like `launch sh -c "htop; read"` do what you want? It should first execute `htop` and when you exit that program, it will execute `read`, which will wait for a keyboard input.

**Comment by kovidgoyal** [comment]
Or launch sh -c "htop; bash"

**Reply by kporkka** [reply]
Thanks both, worked as advised. -kimmo
